pyproject.toml

```diff
@@ -5,11 +5,11 @@ requires = ["setuptools>=61.0"]
 
 # This is configuration specific to the `setuptools` build backend.
 # If you are using a different build backend, you will need to change this.
-[tool.setuptools]
-# define all packages here
-packages = ['causalpy', 'causalpy.data']
```

Comment:
Are you sure about removing this `packages` line? I think this is required.

Reply:
Yes, this now taken care of by `packages.find`. It recursively looks for packages (folders that have an `__init__.py` in them) and includes them into the sdist and wheel distributions. This means that if at a later stage you decide to add another folder to causalpy with special functionality, you won’t have to explicitly add it here, it will get included automatically.
The only thing that I had to do was say that `find` needed to exclude the docs and tests folders, or those would have been included too.